app/migrations/versions/8408aefd8f90_.py

```diff
@@ -0,0 +1,58 @@
+"""empty message
+
+Revision ID: 8408aefd8f90
+Revises: 097d6eedce34
+Create Date: 2021-09-21 04:58:27.495271
+
+"""
```

Comment:
new migration has fixed the 'undefined' when choosing 'Grammar' or 'Research Proposal'. Tested and it worked!